tck/src/main/java/org/eclipse/microprofile/openapi/apps/airlines/resources/ReviewResource.java

```diff
@@ -189,9 +189,7 @@ public Response getReviewByUser(
             description = "username of the user for the reviews",
             required = true,
             in = ParameterIn.PATH,
-            content = @Content(
-                examples = @ExampleObject(
-                    value = "bsmith")))
```

Comment:
can we keep the current `content` field as well?    That way we're not decreasing coverage on other annotations.